libraries/swagger/ConnectorAPI.json

```diff
@@ -1,2640 +1,2674 @@
 {
```

Comment:
This file updated with the lastest version from here:
https://github.com/microsoft/botframework-sdk/blob/master/specs/botframework-protocol/botframework-channel.json